run_brer/plugin_configs.py

```diff
@@ -124,6 +124,8 @@ def build_plugin(self):
         """
         if self.get_missing_keys():
             raise KeyError('Must define {}'.format(self.get_missing_keys()))
+        if self.get('alpha') == 0:
```

Comment:
This looks right. Let's do it in the ProductionPluginConfig, too.

Reply:
Roger, roger.